content/en/docs/tasks/configure-pod-container/configure-persistent-volume-storage.md

```diff
@@ -247,8 +248,8 @@ You can now close the shell to your Node.
 
 You can perform 2 volume mounts on your nginx container:
 
-`/usr/share/nginx/html` for the static website
-`/etc/nginx/nginx.conf` for the default config
+- `/usr/share/nginx/html` for the static website
```

Comment:
I check out the deploy preview and your changes are resulting in the layout of these items being much improved.  Great job!